thewalrus/quantum.py

```diff
@@ -387,12 +387,17 @@ def density_matrix(mu, cov, post_select=None, normalize=False, cutoff=5, hbar=2)
 
     if post_select is None:
         A = Amat(cov, hbar=hbar).conj()
+        sf_order = tuple(chain.from_iterable([[i, i + N] for i in range(N)]))

```

Comment:
This is really nice way of writing this :boom: 
My way is much more complicated